gxcloudstorage-awss3-v2/src/main/java/com/genexus/db/driver/ExternalProviderS3V2.java

```diff
@@ -228,6 +228,10 @@ public void download(String externalFileName, String localFile, ResourceAccessCo
 					while ((read = objectData.read(bytes)) != -1) {
 						outputStream.write(bytes, 0, read);
 					}
+				} catch (IOException e) {
+					logger.error("Error while processing the input stream", e);
+				} finally {
+					objectData.close();
```

Comment:
This is needed? You are inside a Try resource code. Thw clous will be done automatically.